packages/mdc-menu/simple/foundation.js

```diff
@@ -282,7 +281,7 @@ class MDCSimpleMenuFoundation extends MDCFoundation {
     let el = evt.target;
 
     while (el && el !== document.documentElement) {
-      if (this.adapter_.eventTargetHasClass(el, cssClasses.LIST_ITEM)) {
+      if (this.adapter_.getIndexForEventTarget(el) >= 0) {
```

Comment:
Small nitpick: I'd suggest updating the JSDoc description to explain that it cancels if it's not a child item instead of just a list item.

Reply:
Also, perhaps just personal preference, but I prefer checks to be as explicit as possible so it might be a good idea to change the comparator from `>= 0` to `!== -1`